apps/web/next.config.mjs

```diff
@@ -29,4 +31,49 @@ const nextConfig = {
   },
 }
 
-export default nextConfig
+let config
+if (process.env.SENTRY_ORG && process.env.SENTRY_PROJECT) {
+  config = withSentryConfig(nextConfig, {
+    // For all available options, see:
+    // https://github.com/getsentry/sentry-webpack-plugin#options
+
+    org: process.env.SENTRY_ORG,
+    project: process.env.SENTRY_PROJECT,
+
+    // Only print logs for uploading source maps in CI
+    silent: !process.env.CI,
```

Comment:
We don't have this env variable unless is setup automatically by GH actions?

Reply:
it does